### DISCUSSION TITLE
Solar Power Working Group Notes Dec. 22, 2021

### DISCUSSION THREAD

**Discussion by keenanjohnson** [Ribbit-Network/ribbit-network-frog-hardware#90] [Meeting Notes]
**Attendees**
* @eren-rudy 
* @djgood 
* Elias F.
* @keenanjohnson 

**Notes**
* This was the first introductory call with the working group for a solar power Frog Sensor.
* There doesn't seem to be information about what sort of power can we expect? For example, what is the actual performance of a solar cell in a specific area?
* @djgood and @eren-rudy have ordered some parts to start characterizing the performance.
* Instead of thinking about the total Frog System as one big set of requirements, we agreed that it makes sense at least initially to treat the solar-power system as a standalone add on that can be combined with the current Frog Sensor electronics.

** Action Items **
* @djgood and @eren-rudy will share the basic parts that have started with
* Discussion/work will continue in the Github Discussion forums here.

**Comment by eliasfallon** [comment]
Thanks @keenanjohnson - Sorry I dropped off the discussions for a few weeks here, it got real busy and switched back to full remote work. @djgood and @eren-rudy - can you share which hardware you are testing already for the Solar. Then I'll look to get the same thing plus maybe some alternatives, so we can gather a variety of data. Thanks.

**Reply by eren-rudy** [reply]
Hey Elias. We have bought some hardware and have drawn up a circuit, but to be honest it is in a pretty messy state and may just cause more confusion. We plan on cleaning everything up in the next few days and sharing it here with you. Sorry about the delay!

**Reply by eliasfallon** [reply]
No problem, and no hurry. Look forward to seeing it whenever you have it. I'm still trying to get my V2 parts all in and built.
